tide_core.install

```diff
@@ -442,3 +442,26 @@ function tide_core_update_8018() {
   $config->set('display.default.display_options.empty.area.content.value', 'There are no scheduled updates yet.');
   $config->save();
 }
+
+/**
+ * Update Jira workflow notifications message.
+ */
+function tide_core_update_8019() {
+  $config_factory = \Drupal::configFactory();
+  $config = $config_factory->getEditable('tide_workflow_notification.settings');
+  $notification_status = [
+    'draft_needs_review',
+    'draft_published',
+    'needs_review_draft',
+    'needs_review_published',
+    'published_archived',
+  ];
+  $current_string = ['([node:url])', '(https://www.vic.gov.au/how-publish-content-vicgovau).'];
+  $replaced_string = [' - [node:url]', ' https://www.vic.gov.au/how-publish-content-vicgovau'];
+  foreach ($notification_status as $status) {
+    $updated_string = $config->get()['notifications'][$status]['message'];
+    $updated_string = str_replace($current_string, $replaced_string, $updated_string);
+    $config->set('notifications.' . $status . '.message', $updated_string);
```

Comment:
This naming convention hurts my head 😄 I'd like to suggest the following to improve the readability:

```suggestion
    $current_status_message = $config->get()['notifications'][$status]['message'];
    $updated_status_message = str_replace($current_string, $replacement_string, $current_status_message);
    $config->set('notifications.' . $status . '.message', $updated_status_message);
```